doc/index.rst

```diff
@@ -9,100 +7,171 @@
+
+            plot_types/index.rst
+            gallery/index.rst
 
-        - :doc:`Example gallery <gallery/index>`
-        - :doc:`Matplotlib FAQ <users/faq/index>`
 
     .. grid-item-card::
         :padding: 2
+        :columns: 12
 
-        Understand how Matplotlib works
+        **Reference**
         ^^^
 
-        - :ref:`users-guide-explain` in the :doc:`Users guide
-          <users/index>` has a number of advanced topics.
+        .. grid:: 1 1 2 2
+            :class-row: sd-align-minor-center
 
-    .. grid-item-card::
-        :padding: 2
+            .. grid-item::
 
-        Reference
-        ^^^
+                .. toctree::
+                    :maxdepth: 1
+
+                    API reference <api/index>
+                    Figure methods <api/figure_api>
+                    Plotting methods <api/axes_api>
+
+
+            .. grid-item::
 
-        - :doc:`API Reference <api/index>`
-        - :doc:`Axes API <api/axes_api>` for most plotting methods
-        - :doc:`Figure API <api/figure_api>` for figure-level methods
-        - Top-level interfaces to create:
+                Top-level interfaces to create:
 
-          - Figures (`.pyplot.figure`)
-          - Subplots (`.pyplot.subplots`, `.pyplot.subplot_mosaic`)
+                - figures: `.pyplot.figure`
+                - subplots: `.pyplot.subplots`, `.pyplot.subplot_mosaic`
 
+Community
+=========
 
-********************
-Third-party packages
-********************
+.. grid:: 1 1 2 2
+    :class-row: sd-align-minor-center
+
+    .. grid-item::
 
-There are many `Third-party packages
-<https://matplotlib.org/mpl-third-party/>`_ built on top of and extending
-Matplotlib.
+        .. toctree::
+            :maxdepth: 2
 
+            users/resources/index.rst
 
-************
-Contributing
-************
+    .. grid-item::
+
+        :octicon:`link-external;1em;sd-text-info` `Third-party packages <https://matplotlib.org/mpl-third-party/>`_,
+
+        provide custom, domain specific, and experimental features, including
+        styles, colors, more plot types and backends, and alternative
+        interfaces.
+
+What's new
+==========
+
+.. grid:: 1 1 2 2
+
+    .. grid-item::
+
+       Learn about new features and API changes.
+
+    .. grid-item::
+
+        .. toctree::
+            :maxdepth: 1
+
+            users/release_notes.rst
+
+
+Contribute
+==========
+
+.. grid:: 1 1 2 2
+    :class-row: sd-align-minor-center
+
+    .. grid-item::
+
+        Matplotlib is a community project maintained for and by
+        its users.
+
+        There are many ways you can help!
+
+    .. grid-item::
+        .. toctree::
+            :maxdepth: 2
+
+            devel/index.rst
+
+About us
+========
+
+.. grid:: 1 1 2 2
+    :class-row: sd-align-minor-center
+
+    .. grid-item::
+
+        Matplotlib was created by John Hunter to visualize EEG data. John's
+        goal was that Matplotlib make easy things easy and
+        hard things possible.
```

Comment:
If you go into history, I suggest to span the arch to the current state

```suggestion
        Matplotlib was originally created by neurobiologist John Hunter to
        visualize EEG data. It quickly grew into a popular gerneral-purpose
        plotting library. John's goal was that Matplotlib make easy things
        easy and hard things possible.
```

---
On a side note, I still don't like the "easy things easy and hard things possible". The first part is to be expected from any reasonable tool. If you don't do this, you simply have bad user experience. IMHO that's not a mission statement worth mentioning. - It's in particular not great, as matplotlib has a relatively steep initial learning curve (partly due to our heritage) and I'd claim that e.g. plotly makes easy things easier than we do. :(